vpp/clientconfigsrv.go

```diff
@@ -0,0 +1,117 @@
+package vpp
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/satori/go.uuid"
+	"strings"
+)
+
+type ClientContext struct {
+	HostName string `json:"hostname"`
+	Guid     string `json:"guid"`
+}
+
+type Location struct {
+	LocationName string `json:"locationName"`
+	LocationId   int    `json:"locationId"`
+}
+
+type ClientConfigSrvResponse struct {
+	ClientContext      interface{} `json:"clientContext"`
```

Comment:
why is this an empty interface?

Reply:
I changed it to a string.  I then convert that string to a ClientContext struct only when needed.  In this case in the SetClientContext and GetClientContext methods.